server/src/main/java/com/defold/extender/services/DefoldSdkService.java

```diff
@@ -209,6 +212,10 @@ private void deleteCachedSdk(Path path) {
     @PreDestroy
     @SuppressWarnings("unused")
     public void destroy() {
+        if (!this.cacheClearOnExit) {
+            LOGGER.info("Skipping cleanup of SDK cache");
+            return;
+        }
```

Comment:
By not deleting the cache, the local tests should go faster, by not having to download the sdk's all the time.